java/README.md

```diff
@@ -0,0 +1,44 @@
+# Summary
+The Java layer for Vowpal Wabbit uses the [Java Native Interface](https://en.wikipedia.org/wiki/Java_Native_Interface) to communicate between Java and the native Vowpal Wabbit C code.  The philosophy of this layer is that we want to expose a simple type safe, thread safe interface so a Java programmer can use Vowpal Wabbit for either learning or scoring.  An example usage is shown below
+
+```java
+class Foo {
+    static {
+        VWScalarLearner learner = VWLearners.create("-f model.vw"); // Create the model.  The model type is checked here
+        learner.learn("0 | price:.23 sqft:.25 age:.05 2006"); // Learn some data
+        learner.learn("1 2 'second_house | price:.18 sqft:.15 age:.35 1976");
+        learner.learn("0 1 0.5 'third_house | price:.53 sqft:.32 age:.87 1924");
+        learner.close(); // Closing finalizes the model and frees up the native memory
+        
+        VWScalarLearner learner = VWLearners.create("-i model.vw -t --quiet");
+        float prediction = learner.predict("| price:0.23 sqft:0.25 age:0.05 2006"); // Get a prediction.
+        learner.close();
+    }
+}
+```
+
+More examples can be found in the [Java tests](src/test/java/vowpalWabbit/learner).
+
+## Important Notes
+    1.  Most standard Vowpal Wabbit command line options are supported when calling create.  If you find some that work from the regular command line and not within the JNI please file a bug.
```

Comment:
These numbers make a weird code block or something when I view on Github.  Maybe do:

```
1. point 1
1. point 2
1. point 2
```

which should result in:

1. point 1
1. point 2
1. point 2